lavamoat/browserify/policy-override.json

```diff
@@ -164,6 +164,11 @@
         "immer": true
       }
     },
+    "@metamask/keyring-controller": {
+      "packages": {
+        "ulid": true
+      }
+    },
```

Comment:
I wonder if we really needs this override? Since we already have the next one that is strictly related to `"@metamask/keyring-controller>ulid"`?

Has this been added manually?

I've tried to remove it, and it also removes the policy changes (which also seems unnecessary to me, but I could be wrong here..)